lib/wovnrb.rb

```diff
@@ -37,7 +37,9 @@ def call(env)
 
       cookie_lang = Rack::Request.new(env).cookies['wovn_selected_lang']
       request_lang = headers.lang_code
-      if @store.settings['use_cookie_lang'] && cookie_lang.present? && request_lang != cookie_lang && request_lang == @store.default_lang
+      is_get_request = Rack::Request.new(env).get?
```

Comment:
time to make `request = Rack::Request.new(env)` variable at the top

Reply:
Fixed https://github.com/WOVNio/wovnrb/pull/238/commits/a55f6e07fa2e7346ff294e0332fe310947a135f9